pkg/report/sarif_test.go

```diff
@@ -92,6 +93,13 @@ func TestReportWriter_Sarif(t *testing.T) {
 				Schema:  "https://json.schemastore.org/sarif-2.1.0.json",
 				Runs: []*sarif.Run{
 					{
+						Invocations: []*sarif.Invocation{
+							{
+								StartTimeUTC:        &time.Time{},
+								EndTimeUTC:          &time.Time{},
```

Comment:
You can add time to `Report` and here.

Reply:
please let me know if the last three commits address this comment